ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -836,6 +838,13 @@ export default class ConfirmTransactionBase extends Component {
         txData.metadata.note = noteText;
       }
 
+      if (
+        smartTransactionsOptInStatus &&
+        currentChainSupportsSmartTransactions
+      ) {
+        txData.origin += '#smartTransaction';
```

Comment:
Why are we mutating an existing property in this instance as opposed to using a new property for example?

Is the `origin` not traditionally a host name so won't this break that assumption and potentially impact other UX?

Reply:
@matthewwalsh0, we are using the origin because we don’t want to add any new properties for now, as it would impact our Ethereum custodian API.

Reply:
As long as you're sure it won't break any of the UI, fair enough given this branch is specific to MMI.